src/content-helper/editor-sidebar/title-suggestions/provider.ts

```diff
@@ -11,6 +11,8 @@ import {
 	ContentHelperError,
 	ContentHelperErrorCode,
 } from '../../common/content-helper-error';
+import { getLabel as getLabelForTone, ToneProp } from '../../common/components/tone-selector';
+import { getLabel as getLabelForPersona, PersonaProp } from '../../common/components/persona-selector';
```

Comment:
My personal suggestion would be to rename these to `getToneLabel` and `getPersonaLabel` and directly put them into the `addQueryArgs` block, removing lines 48-53. I think that code clarity is also maintained as we would mostly assume we don't want to include smilies in an API request.

Let me know if any of this is unclear, or keep as-is if you think the current solution is better. In this case, I'd suggest to update the wording `humanReadable` to something else.